src/serialization/mod.rs

```diff
@@ -137,7 +144,7 @@
 //! // Make some histograms
 //! for _ in 0..num_histograms {
 //!     let mut h = Histogram::<u64>::new_with_bounds(1, u64::max_value(), 3).unwrap();
-//!     h.record_n(42, 7);
+//!     h.record_n(42, 7).unwrap();
```

Comment:
I really want https://github.com/rust-lang/rfcs/pull/1937 so we can make this use `?`.